src/containers/Cluster/ClusterOverview/components/ClusterMetricsNetwork.tsx

```diff
@@ -0,0 +1,52 @@
+import {DoughnutMetrics} from '../../../../components/DoughnutMetrics/DoughnutMetrics';
+import {formatBytes} from '../../../../utils/bytesParsers';
+import {SHOW_NETWORK_UTILIZATION} from '../../../../utils/constants';
+import {useSetting} from '../../../../utils/hooks/useSetting';
+import i18n from '../../i18n';
+import type {ClusterMetricsBaseProps} from '../shared';
+import {calculateBaseDiagramValues} from '../utils';
+
+import {ClusterMetricsCardContent} from './ClusterMetricsCard';
+
+interface ClusterMetricsNetworkProps extends ClusterMetricsBaseProps {
+    percentsValue: number;
+    throughput?: string;
+}
+
+function formatStorageLegend(value?: string) {
+    return formatBytes({value, withSpeedLabel: true});
+}
+
+export function ClusterMetricsNetwork({
+    percentsValue,
+    throughput,
+    collapsed,
+    ...rest
+}: ClusterMetricsNetworkProps) {
+    const [showNetworkUtilization] = useSetting<boolean>(SHOW_NETWORK_UTILIZATION);
+    if (!showNetworkUtilization) {
+        return null;
+    }
+    const {status, percents, fill} = calculateBaseDiagramValues({
+        fillWidth: percentsValue * 100,
+        ...rest,
+    });
+
+    const legend = formatStorageLegend(throughput);
+
+    return (
+        <ClusterMetricsCardContent
+            status={status}
+            fillWidth={fill}
+            title={i18n('title_storage')}
```

Comment:
In `collapsed` state it is labeled with `Storage` (`title={i18n('title_storage')}`)

<img width="1172" alt="Screenshot 2025-04-22 at 13 59 15" src="https://github.com/user-attachments/assets/c71585bb-ab6e-4052-8fe6-2df914dd9ee7" />